stable/traefik/templates/config-files.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.configFiles }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "traefik.fullname" . }}-configs
+  labels:
+    app: {{ template "traefik.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+{{- range $filename, $fileContents := .Values.configFiles }}
+  {{ $filename }}: {{ $fileContents | b64enc | quote }}
```

Comment:
Willing to be corrected if I am wrong, but I am pretty sure that values in a config map aren't supposed to be base64 encoded.

Reply:
Thanks. Didn't test that one